Library/ViewModels/SelectCurrencyViewModel.swift

```diff
@@ -33,6 +39,51 @@ SelectCurrencyViewModelOutputs {
       initialChosenCurrency,
       self.didSelectCurrencySignal
     )
+
+    let updateCurrencyEvent = self.selectedCurrencyProperty.signal.skipNil()
+      .takeWhen(self.saveButtonTappedSignal.ignoreValues())
+      .switchMap { input in
+        AppEnvironment.current.apiService
+          .changeCurrency(input: ChangeCurrencyInput(chosenCurrency: input.rawValue))
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
```

Comment:
Any reason for this delay? I know we sometimes use it, sometimes we don't.

Reply:
Yeah we use `ksr_delay` so that we can do an `self.scheduler.advance()` in the tests. I'm doing this in #572 and it's here because I broke the work up retroactively for the multiple PR's.